src/common/translations/localize.ts

```diff
@@ -89,7 +89,7 @@ export const computeLocalize = async <Keys extends string = LocalizeKeys>(
   resources: Resources,
   formats?: FormatsType
 ): Promise<LocalizeFunc<Keys>> => {
-  await import("../../resources/intl-polyfill").then(() =>
+  await import("../../resources/polyfills/intl-polyfill").then(() =>
     polyfillLocaleData(language)
```

Comment:
Do we really need this here?  Is there any situation where `language` would be different from that returned by `getLocalLanguage` within the polyfill?

Reply:
NM, this seems to be the only place we add data if the user changes language.  We should move it closer to that trigger because here it won't help anything that renders without calling `localize`.

Anyway, I improved the bundles by just lazily importing `intl-messageformat` instead here, which reduces the entry point sizes.